src/Tribe/Tickets_Handler.php

```diff
@@ -124,6 +124,47 @@ public function print_checkedin_totals() {
 	}
 
 	/**
+	 * Get the total capacity event.
+	 *
+	 * @param $event_id int (null)
+	 *
+	 * @since TBD
+	 *
+	 * @return int number of tickets ( -1 means unlimited )
+	 */
+	public function get_total_event_capacity( $event_id = null ) {
+		if ( null === $event_id ) {
+			global $post;
+			$event_id = $post->ID;
```

Comment:
let's not use event terminology because we can attach Tickets to any post.
Also it would be better to allow either the int or the `WP_Post` obj.
```php
// If post is not numeric or instance of WP_Post it defaults to the current Post ID
if ( ! is_numeric( $post ) && ! $post instanceof WP_Post ) {
	$post = get_the_ID();
}

// If not a WP_Post we try to fetch it as one
if ( is_numeric( $post ) ) {
	$post = WP_Post::get_instance( $post );
}

// Prevent Non usable $post instances
if ( ! $post instanceof WP_Post ) {
	return null;
}

```